From: Virendra Pathak <virendra.pathak@broadcom.com>
Date: Mon, 20 Jun 2016 09:26:43 +0100
Subject: [PATCH] Update the feature set for the Vulcan AArch64 cpu.

gas	* config/tc-aarch64.c (aarch64_cpus): Update vulcan feature set.

Index: binutils-2.26/gas/config/tc-aarch64.c
===================================================================
--- binutils-2.26.orig/gas/config/tc-aarch64.c
+++ binutils-2.26/gas/config/tc-aarch64.c
@@ -7774,8 +7774,8 @@ static const struct aarch64_cpu_option_t
   {"thunderx", AARCH64_FEATURE (AARCH64_ARCH_V8,
 				AARCH64_FEATURE_CRC | AARCH64_FEATURE_CRYPTO),
    "Cavium ThunderX"},
-  {"vulcan", AARCH64_FEATURE (AARCH64_ARCH_V8,
-			      AARCH64_FEATURE_CRC | AARCH64_FEATURE_CRYPTO),
+  {"vulcan", AARCH64_FEATURE (AARCH64_ARCH_V8_1,
+			      AARCH64_FEATURE_CRYPTO),
   "Broadcom Vulcan"},
   /* The 'xgene-1' name is an older name for 'xgene1', which was used
      in earlier releases and is superseded by 'xgene1' in all
